lmms_eval/tasks/mmsi_bench/utils.py

```
@@ -1,13 +1,14 @@
+import io
 import logging
 import re
-from PIL import Image
+from collections import defaultdict
+
 import numpy as np
```

Ruff (0.11.9)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)